src/GafferScene/OpenGLAttributes.cpp

```diff
@@ -78,10 +78,11 @@ OpenGLAttributes::OpenGLAttributes( const std::string &name )
 	attributes->addChild( new NameValuePlug( "gl:curvesPrimitive:glLineWidth", new IECore::FloatData( 1.0 ), false, "curvesPrimitiveGLLineWidth" ) );
 	attributes->addChild( new NameValuePlug( "gl:curvesPrimitive:ignoreBasis", new IECore::BoolData( false ), false, "curvesPrimitiveIgnoreBasis" ) );
 
-	// light visualisers
-
-	attributes->addChild( new Gaffer::NameValuePlug( "gl:visualiser:maxTextureResolution", new IECore::IntData( 512 ), false, "maxTextureResolution" ) );
+	// visualisers
 
+	attributes->addChild( new Gaffer::NameValuePlug( "gl:visualiser:ornamentScale", new IECore::FloatData( 1.0f ), false, "visualiserOrnamentScale" ) );
```

Comment:
Same here - would be nice to enforce some limits.

Reply:
https://github.com/GafferHQ/gaffer/pull/3519/commits/3b334830f74592fb556b71f91e8d68afba5a9bd1